src/dotnet/commands/dotnet-projectmodel-server/ProjectManager.cs

```diff
@@ -122,10 +122,8 @@ private void ProcessLoop()
             }
             catch (Exception ex)
             {
-                // TODO: review error handing logic
-
-                _log.LogError($"Error occurred: {ex}");
-
+                _log.LogError("Error occurs: {0}", ex.ToString());
```

Comment:
Should we be handling the FileFormatException specially and producing a more descriptive log message?